inworld-ai/inworld-unity-sdk
# CORE-4829 Add Websocket support for session state

![image](https://github.com/inworld-ai/inworld-unity-sdk/assets/68676221/7ff08095-8363-4793-b3cf-d5eeeea434bd)

The history item in this demo will generated in the runtime only and not be saved. Developers needs to create their own method to store the data.

How to test:
1. New Game.
2. Say something
3. Save Game
4. Quit Game
5. Load Game
6. Say something again to test if the character can remember.

NOTE: The character may not respond to you due to its own personality. If it hides the answer, please check the string of LoadSceneResponse, if it contains in the data or stateHolder.

TODO: Update NDK dll to make it compatible (For NDK, only previousState is worked)
